src/user-federation/ldap/mappers/LdapMapperRoleGroup.tsx

```diff
@@ -61,7 +62,7 @@ export const LdapMapperRoleGroup = ({
     },
     (fetchedComponent) => {
       if (fetchedComponent) {
-        const vendor = fetchedComponent.config?.vendor[0];
+        setVendor(fetchedComponent.config?.vendor[0]);
```

Comment:
`vendor` is undefined on the initial load so the defaultValue always gets set to `groupOfNames`
```suggestion
        const vendor = fetchedComponent.config?.vendor[0];
        setVendorType(vendor);
```

Reply:
thanks @jenny-s51, good catch 👍  - this wouldn't have worked as originally coded, i thought i had tested that. should be fixed now.